page_templates/style_guide.md

```diff
@@ -0,0 +1,55 @@
+# Jina AI Documentation Style Guide
+
+Guides are publicly available and represent the Jina AI brand and the quality of the product. Please adhere to the following steps to ensure basic professionalism in the docs.
+
+## Guiding principles:
+
+-   **Accessibility**. Everyone who needs to should understand our documents.
+
+-   **Professional**. A consistent writing and formatting style will give a professional look to the documents.
+
+## Specific guidelines:
+
+**[ For anything not covered here, we default to the [Google developer style guide.](https://developers.google.com/style)  ]**
+
+1.  Check the spelling and grammar in your contributions. Most editors include a spell checker or have an available spell-checking plugin. You can also paste your text into a Google Doc or other document software for a more robust spelling and grammar check. We use American English spelling.
+
+2.  Avoid run-on sentences. Generally, keep sentences to less than 13 words longs.
+
+3.  When explaining technical steps, use a numbered step by step process. Don't list out every step in a single paragraph.
+
+4.  When explaining general concepts, paragraphs should be used. Yet paragraphs should not be over six sentences long.
+
+5.  Use the second person tense and describe actions to the user. (this is the same as active style or imperative style)
+
+> You should open Jina using python.
+> In your pod, you will find.
+
+6.  When using Jina terms, they should be Capitalized. Flow, Pod, JinaD. Anything found in the Jina glossary or Jina 101. Including Executor types (Evaluator, Segmenter, etc.)
+
+7. Don't capitalize Flow types (index, query). Don't capitalize the adjective or  `-tion`  form (i.e. evaluation, evaluating, segmenting, etc)
+
+8.  Use YAML, not yaml.
```

Comment:
and JSON not json. Also
- API
- gRPC
- REST
- RESTful (we don't use this afaik but just in case)
- ID not id?

Reply:
Woop, good question. [Apple](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/APIAbbreviations.html) seems to suggest that certain abbreviations that are well known do not need to be explained. I think we'll just to have play this one by ear. 

I've updated point 8, let me know what you think of it

Reply:
Good seems to suggest using a [wordlist](https://developers.google.com/style/word-list), which I like the idea of it. I added one at the bottom for these cases.